### DISCUSSION TITLE
Mini problems with Front page, Figures and Bibliography

### DISCUSSION THREAD

**Discussion by Matilde-rod** [joaomlourenco/novathesis#519] [Q&A] [answered]
Good afternoon :)

I'm in the final stage of editing my dissertation for DEMI of FCT-NOVA and have some doubts:

1- On the front page, my department, DEMI, breaks one of the words in half. Also, I would like to change from master to integrated master and don't know how. This being said, I know there is the 9_fct file, but I couldn't make it work so I just wrote in the normal frontpage file, maybe there I can make the alterations I need?
![demi](https://github.com/user-attachments/assets/beedbb1d-811e-4100-b31d-1cef2c33c48a)
![master](https://github.com/user-attachments/assets/84082e60-871a-4c3b-b92a-186bace9d4e8)

2- My professor asked if I could take the references of the images and tables from the context index, i really don't know if this is possible, to keep the caption  with reference but not in the index caption

3- I have one reference where it only shows the name and not the rest of the entry. And I have another that is crossing the page margins.
![qui](https://github.com/user-attachments/assets/00612ffc-0a6d-4fb5-b36f-cae5b7f80f82)
![xuanqui](https://github.com/user-attachments/assets/da4ab19e-68b5-4fd9-bc82-e6093c366826)
![refcross](https://github.com/user-attachments/assets/c75dd496-1962-4f5f-a6b1-082e8bb5a5b2)

I would be grateful for any help  :)
Thanks!!

**Comment by joaomlourenco** [comment, marked as answer]
1) Open the `9_nova_fct.tex`file and uncomment the department and degree name. That's all.

2) Use `\capition[Caption for the list of figures/tables.]{Real caption for the figure/table.}`

3) I can't see/download your images, but there must be some error in the entry that does not print correctly — check for missing quotes ("), braces (}) and commas (,).  For the URLs, always use "url = {\url{blah}},"

**Reply by Matilde-rod** [reply]
Thanks for the answer!
The refs i could figure out but the rest, i still can't make it work  :///
The \caption is what i've been using since the beginning for the caption of figures and tables, but the name I put there is what shows in the index, it doesn't take the ref out.
About the frontpages, i've tried to use the 9_nova_fct but it always gives me the undefined error on the lines i uncomment.
Are there any other solutions I can try?

**Reply by joaomlourenco** [reply]
1) great.

2) Please don't be light headed and follow my suggestion above!  If you do, it will work!

3) I don't know what is your problem. It should definitely work.  What error do you get?

**Reply by Matilde-rod** [reply]
Indeed was being light headed, it worked!

3. When i run the 9, i get this 

9_nova_fct.tex
55
Undefined control sequence.
l.55  \ntdepartment
                   (pt){Departamento de\\Engenharia Mecânica e Industrial}
                
(And the same on degree)
And when i run in the template nothing happens, it doesn't disapear or change, it keeps the same as when i uncoment in the cover file

**Reply by joaomlourenco** [reply]
You don't “run the 9".   Just uncomment the right lines and recompile the whole template.

**Reply by Matilde-rod** [reply]
Okok then it doesn't make the changes i need :/
It keeps only master and not integrated and still cuts the word of the department (This last only happens on the 2º cover page, the front one keeps it intact)

**Reply by joaomlourenco** [reply]
To have the "Integrated Master in" add to file `Config/5_packages.tex`:
```latex
degreenameprefix(msc,pt):={Mestrado Integrado em}
degreenameprefix(msc,en):={Integrated Master in}
```

Concerning the department name, it works for me!  So, I don't know what is your problem… :(
<img width="588" alt="image" src="https://github.com/user-attachments/assets/61c238a8-e783-4cff-b0eb-0626f0eb6c88">

**Reply by Matilde-rod** [reply]
Thank you for the help! :)

**Reply by joaomlourenco** [reply]
Once you finish your work, please remember to send a small donation following the link at the top of the main page.  Thanks.

**Reply by Jasolt7** [reply]
I've tried adding to the 0-Config/5_packages.tex:
degreenameprefix(msc,pt):={Mestrado Integrado em}
degreenameprefix(msc,en):={Integrated Master in}

However I get an error on the same file:
! LaTeX Error: Missing \begin{document}.
See the LaTeX manual or LaTeX Companion for explanation.
Type H <return> for immediate help.
...
l.41 d
egreenameprefix(msc,pt):={Mestrado Integrado em}

**Reply by joaomlourenco** [reply]
There is a backslash missing. 

`\degreenameprefix(msc,pt):={Mestrado Integrado em}`
